.github/workflows/module-tests.yml

```diff
@@ -52,24 +52,6 @@ jobs:
             ./mvnw test -B -Dair.check.skip-all -pl presto-hive -P test-hive-glue
           fi
 
-  kudu-tests:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: 8
-      - name: Install Kudu Module
-        run: |
-          export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./bin/retry ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl presto-kudu
-      - name: Run Kudu Tests -- disable InferSchema
-        run: presto-kudu/bin/run_kudu_tests.sh "null"
-      - name: Run Kudu Tests -- enable InferSchema, empty prefix
-        run: presto-kudu/bin/run_kudu_tests.sh ""
-      - name: Run Kudu Tests -- enable InferSchema, standard prefix
-        run: presto-kudu/bin/run_kudu_tests.sh "presto::"
```

Comment:
It looks like we are now losing some tests coverage. Not sure how much important is this. I wonder if it is possible to have some smoke tests to be executed 3 times for these cases, like having base smoke tests class for kudu and three different implementation with three different `KuduServer`. That requires that `KuduServer` need to be parametric.